res/TensorFlowLiteRecipes/Net_Maximum_Minimum_000/test.recipe

```diff
@@ -0,0 +1,86 @@
+operand {
+  name: "Const"
+  type: FLOAT32
+  shape {
+  }
+  filler {
+    tag: "explicit"
+    arg: "6"
+  }
+  quant {
+    quantized_dimension: 0
+  }
+  is_variable: false
+}
+operand {
+  name: "Const_1"
+  type: FLOAT32
+  shape {
+  }
+  filler {
+    tag: "explicit"
+    arg: "0"
+  }
+  quant {
+    quantized_dimension: 0
+  }
+  is_variable: false
+}
+operand {
+  name: "Hole"
+  type: FLOAT32
+  shape {
+    dim: 1
+    dim: 16
+    dim: 160
+    dim: 160
```

Comment:
Is this NCHW form?  Would be better to go with NHWC
And if we run this in our interpreter, would take some times, so it would be better to go with small values like 1,4,4,3